docs/dns-server.md

```diff
@@ -31,8 +31,8 @@ Tested on Debian GNU/Linux 9.6 (stretch).
 
 ```Shell
 sudo apt -y install resolvconf dnsmasq
-sudo sh -c 'echo "address=/.vm/127.0.0.1" >> /etc/NetworkManager/dnsmasq.d/development'
-sudo sh -c 'echo "address=/.vm23.iveins.de/127.0.0.1" >> /etc/NetworkManager/dnsmasq.d/development'
+sudo sh -c 'echo "address=/.vm/127.0.1.1" >> /etc/dnsmasq.d/development'
```

Comment:
Isn't that change specific to your setup?
What if my virtual box has a different local-ip-setup, will that change render useless for my hypothetical machine?

Reply:
This is only practical if you have a fake domain.

For a real domain like vm.company.org, you do not need that. But it can be handy that you can also work without internet. Of course, there can also be the /etc/hosts temporary help.

Should it be more adapted to the company or towards the community?